app/assets/stylesheets/custom.scss

```diff
@@ -128,16 +112,53 @@ a {
 
 #store-name {
   color: #504d4f;
+  float: left;
+  font-size: 1.6em;
+  margin: auto;
 }
 
 .retire-btn {
   margin-bottom: 54px;
 }
 
+#complete-your-loan {
+  text-align: center;
+}
+
 td {
   background-color: rgba(255, 255, 255, .5);
 }
 
+// flash alerts
+#flash_error {
```

Comment:
Avoid using id selectors<br>Selector `flash_error` should be written in lowercase with hyphens